### ISSUE TITLE
enemy keep lord quest

### ISSUE THREAD

**Issue by Xicoz** [WarEmu/WarBugs#15393]
in the enemy keeps, you can get the "defend keep" by clicking the enemy lord.
![Betamale_014](https://user-images.githubusercontent.com/44329952/71763828-32cd8500-2ea6-11ea-9750-9d656c2cc464.jpg)

**Comment by raptrr** [comment]
Secrets could not reproduce it but will check later again.
Can you confirm it on any other keep lords?

**Comment by Xicoz** [comment]
yes, in it has happened to me in DW as well, but those are the only 2 keeps ive noticed, I will also try to reproduce when we take a keep again

**Comment by Xicoz** [comment]
just reproduced again, took some spam clicking but I got the quest
![Betamale_015](https://user-images.githubusercontent.com/44329952/71770870-f3c81f80-2ef7-11ea-9757-3d1d589f5e33.jpg)

**Comment by Xicoz** [comment]
![Nooblit_004](https://user-images.githubusercontent.com/44329952/71773404-1d963c00-2f22-11ea-8173-2e585895310d.jpg)
